xep-0070.xml

```diff
@@ -348,6 +348,9 @@ Content-Length: 3032
   <section2 topic='End-to-End Encryption' anchor='security-e2e'>
     <p>For added security, the XMPP Server and XMPP Client may wish to communicate using end-to-end encryption. Methods for doing so are outside the scope of this proposal.</p>
   </section2>
+  <section2 topic='Misleading password prompt' anchor='security-password'>
+    <p>The default login prompt in most web browsers will ask the user for a password where they should instead enter a transaction identifier in the case of XMPP authentication. This may lead users to accidentally enter their XMPP account password as a transation identifier. Websites using this authentication method in an user facing way SHOULD NOT use the standard web browser login dialog, and let the user enter their credentials by other means.</p>
```

Comment:
```suggestion
    <p>The default login prompt in most web browsers will ask the user for a password where they should instead enter a transaction identifier in the case of XMPP authentication. This may lead users to accidentally enter their XMPP account password as a transation identifier. Websites using this authentication method in an user facing way should avoid the standard web browser login dialog, and let the user enter their credentials by other means.</p>
```